packages/test-utils/make-mount.ts

```diff
@@ -0,0 +1,8 @@
+import { mount } from '@vue/test-utils'
+import { merge } from 'lodash'
+
+const makeMount = <C,O, E>(element: C, defaultOptions: O) => {
```

Comment:
Shoulda add a space after the comma.

Reply:
I don't think the function covers most use case in the code base, though.

Reply:
Agreed, we have no idea what situation in the future we might encounter, let's keep this function evolving.